L1Trigger/TrackFindingTracklet/interface/PurgeDuplicate.h

```diff
@@ -40,19 +40,15 @@ namespace trklet {
     bool isSeedingStub(int, int, int);
     std::string l1tinfo(const L1TStub*, std::string);
     std::pair<int, int> findLayerDisk(const Stub*);
-    std::vector<double> getInventedCoords(
-        unsigned int,
-        const Stub*,
-        Tracklet*);  // calculate stub coordinates based on tracklet trajectory for prompt tracking
-    std::vector<double> getInventedCoordsExtended(
-        unsigned int,
-        const Stub*,
-        Tracklet*);  // calculate stub coordinates based on tracklet trajectory for extended tracking
-    std::vector<const Stub*> getInventedSeedingStub(
-        unsigned int,
-        Tracklet*,
-        std::vector<const Stub*>);  // return stub with invented x,y,z coords, if it's a seeding one for this tracklet
-
+    // calculate stub coordinates based on tracklet trajectory for prompt tracking
+    std::vector<double> getInventedCoords(unsigned int, const Stub*, Tracklet*);
+    // calculate stub coordinates based on tracklet trajectory for extended tracking
+    std::vector<double> getInventedCoordsExtended(unsigned int, const Stub*, Tracklet*);
+    // return stub with invented x,y,z coords, if it's a seeding one for this tracklet
+    std::vector<const Stub*> getInventedSeedingStub(unsigned int, Tracklet*, std::vector<const Stub*>);
+    unsigned int findVarRInvBin(Tracklet* trk);
+    std::vector<unsigned int> findOverlapRInvBins(Tracklet* trk);
+    bool findBin(std::vector<unsigned int> vec, unsigned int num);
     std::vector<Track*> inputtracks_;
```

Comment:
Introduce a line space between the last function and first data member (if auto style tools allow it ...).

Reply:
Line space has been introduced, and the auto style tool seems to allow it.